vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -56,41 +57,98 @@ import (
 	google_protobuf "google.golang.org/cloud/library/proto/google/protobuf"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
 )
 
 const (
-    prodAddr = "library-example.googleapis.com:443"
-)
-
-var (
-    allScopes = []string {
-        "https://www.googleapis.com/auth/library",
-        "https://www.googleapis.com/auth/cloud-platform",
-    }
+    gapicNameVersion = "gapic/0.1.0"
 )
 
 var (
     shelfPathTemplate = gax.MustCompilePathTemplate("shelves/{shelf}")
     bookPathTemplate = gax.MustCompilePathTemplate("shelves/{shelf}/books/{book}")
 )
 
-func idempotentRetryCodes() []gax.CallOption {
-    return []gax.CallOption{
-        gax.WithRetryCodes([]codes.Code{
-            codes.DeadlineExceeded,
-            codes.Unavailable,
-        }),
+func defaultClientSettings() gax.ClientSettings {
+    return gax.ClientSettings{
+        AppName: "gax",
+        AppVersion: gax.Version(),
+        Endpoint: "library-example.googleapis.com:443",
+        Scopes: []string{
+            "https://www.googleapis.com/auth/library",
+            "https://www.googleapis.com/auth/cloud-platform",
+        },
+        CallSettings: map[string][]gax.CallOption{
+            "CreateShelf": []gax.CallOption{
+                gax.WithRetryCodes(nonIdempotentRetryCodes()),
+                defaultRetryOptions()...
```

Comment:
I think this is invalid syntactically. Instead, probably
```
append([]gax.CallOption(gax.WithRetryCodes(nonIdempotentRetryCodes()), defaultRetryOptions()...)
```
?

See the compile errors of https://play.golang.org/p/HptLq_gC7j FYI.